packages/clerk-js/src/ui/hooks/useFetchRoles.ts

```diff
@@ -13,7 +13,10 @@ const getRolesParams = {
 };
 export const useFetchRoles = (enabled = true) => {
   const { organization } = useOrganization();
-  const { data, isLoading } = useFetch(enabled ? organization?.getRoles : undefined, getRolesParams);
+  const { data, isLoading } = useFetch(enabled && !!organization?.id ? organization?.getRoles : undefined, {
+    ...getRolesParams,
+    orgId: organization?.id,
+  });
```

Comment:
🙃 Do we populate the `organization` instance for personal account? If not i would expect to use `enabled && !!organization ` instead. Eg:
```typescript
  const { data, isLoading } = useFetch(enabled && !!organization ? organization?.getRoles : undefined, {
    ...getRolesParams,
    orgId: organization?.id,
  });
```